index.yaml

```diff
@@ -20,6 +41,6 @@ entries:
     name: sombra
     type: application
     urls:
-    - https://transcend-io.github.io/sombra-helm-chart/sombra-0.3.1.tgz
+    - https://transcend-io.github.io/helm-chart/sombra-0.3.1.tgz
```

Comment:
this URL is also off, I'd suggest changes but i think this is a generated file so you may have to re-gen